Downloaders/Kemono.party/README.MD

```diff
@@ -4,4 +4,8 @@ Currently supports Patreon, Fanbox, Subscribestar and Gumroad content.
 
 Separate GuGs are used for each site.
 
-The site still occasionally gives errors, so be prepared to check for that stuff.
\ No newline at end of file
+The site still occasionally gives errors, so be prepared to check for that stuff.
+
+##Note
```

Comment:
Markdown on GitHub requires a space after the header hash marks.

```suggestion
## Note
```

Reply:
Yep, I knew that, I just did a stupid.